subsys/settings/src/settings_priv.h

```diff
@@ -17,20 +17,6 @@
 extern "C" {
 #endif
 
-int settings_cli_register(void);
-int settings_nmgr_register(void);
-
-struct mgmt_cbuf;
-int settings_cbor_line(struct mgmt_cbuf *cb, char *name, int nlen, char *value,
-		       int vlen);
-
-void settings_line_io_init(int (*read_cb)(void *ctx, off_t off, char *buf,
```

Comment:
this one, contrary to the others seems to be duplicated and not just dead....

Reply:
Fixed commit mesasge.